hack/make/ubuntu

```diff
@@ -150,6 +160,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
See https://github.com/docker/docker/blob/ac7a33b45ce5061079226a254e1fe6173d3fbd81/hack/make/.build-deb/rules#L7 where this happens in the new packages.  It's only `Recommends` for Ubuntu, where Docker literally won't work without the AppArmor package since AppArmor is installed and configured by default.  Adding this as `Recommends` in this hacky FPM package has way too much potential to break people's existing setups by partially or even fully enabling AppArmor for them.